java/client/src/main/java/glide/api/models/RedisValue.java

```diff
@@ -0,0 +1,32 @@
+package glide.api.models;
+
+import java.util.Map;
+import lombok.Getter;
+
+/** A union-like type which can store single or bulk value retrieved from Redis. */
+@Getter
+public class RedisValue {
+    /** Get per-node value. */
+    private Map<String, Object> multiValue = null;
```

Comment:
In some cases, we should know what the return type would be.  For example, for CONFIG GET, we know we get a Map back from Redis for each node.  
The single value would also be a Map.  We should include a type in this object that defines the return type for single Value or the value type of the map for multiValue.

Reply:
Consider the work we put together under: 
https://github.com/Bit-Quill/glide-for-redis/compare/java/integ_acarbo_api_baseCommands...Bit-Quill:glide-for-redis:java/dev_acabo_cluster_rework_with_examples

Reply:
Reworked, `RedisValue` has a type parameter as of now